app/src/main/java/org/konkuk/placelist/main/MapFragment.kt

```diff
@@ -114,7 +114,11 @@ class MapFragment : Fragment(), OnMapReadyCallback {
             mMap.clear()
             marker = mMap.addMarker(markerOptions.position(it))
             mMap.addCircle(circleOptions.center(it).radius(radius))
-            mMap.moveCamera(CameraUpdateFactory.newLatLngZoom(marker!!.position, 15.0f))
+            if(mMap.cameraPosition.zoom>=15.0f) {
+                mMap.moveCamera(CameraUpdateFactory.newLatLngZoom(marker!!.position, mMap.cameraPosition.zoom))
+            }else {
+                mMap.moveCamera(CameraUpdateFactory.newLatLngZoom(marker!!.position, 15.0f))
+            }
```

Comment:
축소한 뒤에도 유지할 수 있도록 설정해주세요~! 확대 시에만 유지되고 축소 시에는 유지되지 않습니다
최초 구동 시에만 줌되도록 하는 `isCameraZoomChanged` 변수를 두고, 해당 변수를 카메라 움직일 때 `true`로 바꾸는 건 어떨까요

```suggestion
            mMap.moveCamera(CameraUpdateFactory.newLatLngZoom(marker!!.position, if (isCameraZoomChanged) mMap.cameraPosition.zoom else 15.0f))
            isCameraZoomChanged = true
```